Sources/xcproj/PBXProductType.swift

```diff
@@ -21,6 +21,7 @@ public enum PBXProductType: String, Decodable {
     case stickerPack = "com.apple.product-type.app-extension.messages-sticker-pack"
     case xpcService = "com.apple.product-type.xpc-service"
     case ocUnitTestBundle = "com.apple.product-type.bundle.ocunit-test"
+    case xcodeExtension = "com.apple.product-type.xcode-extension"
```

Comment:
I think somehow you got in this branch changes that were merged into master. You should branch off from release/2.0.0 rather than master.